.github/workflows/release_event.yml

```diff
@@ -0,0 +1,17 @@
+name: Release Dispatch Event
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  Notify-of-New-Release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Repository Dispatch
+        uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.DISPATCH_ACCESS_TOKEN }}
```

Comment:
This secret must be added to the repository. It must be a personal access token with the repo scope. I have created one in my own GH account which we can use for testing, but the token used in perpetuity should be associated with an organization-level GH account (if we have one).

I don't have permissions to add secrets to the repo, so someone with admin privileges will have to do so on my behalf. Reach out to me via Slack for the token value.

Reply:
> the token used in perpetuity should be associated with an organization-level GH account (if we have one)

@NevilleS do we have one of these?

Reply:
We don't have any such account right now.